snippets.json

```diff
@@ -0,0 +1,90 @@
+{
+    "For Loop": {
+        "prefix": [
+            "for"
+        ],
+        "body": [
+            "var ${1:i} = 0;",
```

Comment:
This one looks a bit too specialized, why declare both i and j and increment i by j?
```
for (... in Iter.range(...,...)) {
  ...
};
```
or even just
```
for (... in ...) {
  ...
};
```
might be all you want in this.

(The former won't be well-typed unless Iter is imported...)